app/mailers/submission_mailer.rb

```diff
@@ -11,11 +11,11 @@ def new_submission_confirmation_email(submission)
 
     @form_name = @submission.form_name
     if @form_name.downcase.include?("ask")
-      @form_contact = current_organization.ask_form_contact
+      @form_contact = Organization.current_organization.ask_form_contact
```

Comment:
change to model method since @current_organization isn't available in the mailer